repository-quickfix/src/main/java/io/fixprotocol/orchestra/quickfix/DataDictionaryGenerator.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2017-2020 FIX Protocol Ltd
+ * Copyright 2017-2021 FIX Protocol Ltd
```

Comment:
@donmendelson IANAL, but I guess this needs to be updated to the current year?

Reply:
ok